### ISSUE TITLE
Move caesar cypher interactive

### ISSUE THREAD

**Issue by hayleyavw** [uccser/cs-field-guide#15]
Move interactive from Encryption Coding chapter
```
<div class="interactive row">
<div class="col-xs-12 col-sm-4">  
<h4>Plaintext</h4>
<textarea id="caesar-plaintext" class="form-control" rows="5"></textarea>
<button id="caesar-encrypt" type="button" class="btn btn-primary btn-block" style="margin-top:10px;">Encrypt</button>
</div>
<div class="col-xs-12 col-sm-4">
<h4>Rotation Amount</h4>
<input id="rotation-input" type="number" min="1" max="26" value="1" onkeydown="return isNumber(event,this);" class="interactive-input">
</div>
<div class="col-xs-12 col-sm-4">
<h4>Ciphertext</h4>
<textarea id="caesar-ciphertext" class="form-control" rows="5"></textarea>
<button id="caesar-decrypt" type="button" class="btn btn-primary btn-block" style="margin-top:10px;">Decrypt</button>
</div>
</div>
```

**Comment by Heidirose** [comment]
Pretty sure @Jamesernator has done this one?

**Comment by timbellnz** [comment]
The Caesar interactive in the encryption chapter doesn't currently work properly for me. Should it? It currently will encode one text (or decode one text), but can't decode or do a new encoding until the page is reloaded.

**Comment by JackMorganNZ** [comment]
This was fixed in 2.0.1
